posthog/models/event/query_event_list.py

```diff
@@ -160,12 +160,22 @@ def run_events_query(
     person_id = query.personId
     order_by = query.orderBy
     select = query.select
-    where = query.where
+    where = []  # make a new object since we'll be modifying it
+    if query.where:
+        where.extend(query.where)
     event = query.event
 
+    all_properties = []
+    all_properties.extend(query.fixedProperties or [])
+    all_properties.extend(query.properties or [])
+
+    # Split HogQL properties from the rest, as "where" supports filtering by "having" aggregations like "count() > 2"
     properties = []
```

Comment:
`all_properties` vs `properties` is a bit confusing. Maybe this should be `classic_properties` or `non_hogql_properties`?